src/main/java/com/google/api/gax/core/RetrySettings.java

```diff
@@ -203,14 +204,15 @@ public Builder toBuilder() {
     public abstract double getRpcTimeoutMultiplier();
 
     /**
-     * See the class documentation of {@link RetrySettings} for a description
-     * of what this value does.
+     * The MaxRpcTimeout puts a limit on how much the RPC timeout can be increased by the
+     * RpcTimeoutMultiplier.
      */
     public abstract Duration getMaxRpcTimeout();
 
     /**
-     * See the class documentation of {@link RetrySettings} for a description
-     * of what this value does.
+     * The TotalTimeout parameter has ultimate control over how long the logic should keep trying
+     * the remote call until it gives up completely. The higher the total timeout, the more retries
+     * can be attempted.
      */
     public abstract Duration getTotalTimeout();
```

Comment:
Make the ordering of the concepts in the builder the same as the immutable class.